api/src/search/backend/load_opportunities_to_index.py

```diff
@@ -274,7 +293,20 @@ def load_records(self, records: Sequence[Opportunity]) -> set[int]:
                 self.increment(self.Metrics.TEST_RECORDS_SKIPPED)
                 continue
 
-            json_records.append(schema.dump(record))
+            json_record=schema.dump(record)
+
+            json_record["attachments"] = []
+            attachments = self.fetch_attachments(record.opportunity_id)
+
+            for attachment in attachments:
+                file_loc, file_name = attachment
+                # data = self.get_attachment_data(file_loc)
+                json_record["attachments"].append({
+                    "filename": file_name ,
+                    "data": "data"
+                })
+
+            json_records.append(json_record)
```

Comment:
I'd suggest that the method we call just return the JSON objects we'll want. So this should look like:

```
json_record=schema.dump(record)

if <whatever check we add for an env var>:
     json_record["attachments"] = get_attachment_json_for_opportunity(...)

json_records.append(json_record)
```